sim/shaman/fire_totems.go

```diff
@@ -30,7 +30,7 @@ func (shaman *Shaman) registerSearingTotemSpell() {
 			},
 		},
 
-		BonusHitRating:   float64(shaman.Talents.ElementalPrecision) * 1 * core.SpellHitRatingPerHitChance,
```

Comment:
I just wanted to let you know that all the spots that have `x * 1` was not because I'm a crazy person, the idea was to explicitly show the talents multiplier. Sometimes you'll get like 2% per talent point. If it was 1% per talent point, I would leave the explicit *1

I'm not against you removing them because it is less code, but there was a reason for it 🙂

Reply:
Ye, I've thought about that when removing them. I prefer the shorter version, so I went with that.